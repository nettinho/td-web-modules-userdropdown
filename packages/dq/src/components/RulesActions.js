import React from "react";
import { Link } from "react-router-dom";
import { Button } from "semantic-ui-react";
import { FormattedMessage } from "react-intl";
import { linkTo } from "../routes";
import ExecuteRuleButton from "./ExecuteRuleButton";

const createRuleUrl = linkTo.RULE_NEW;

export const RulesActions = () => (
  <div style={{ float: "right" }}>
    {/* <Button
      style={{ marginBottom: "10px" }}
      secondary
      as={Link}
      to={createRuleUrl()}
      content={<FormattedMessage id="quality.actions.create" />}
    /> */}
    <ExecuteRuleButton />
  </div>
);

export default RulesActions;
