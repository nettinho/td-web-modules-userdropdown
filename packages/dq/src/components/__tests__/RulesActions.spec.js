import React from "react";
import { shallow } from "enzyme";
import { RulesActions } from "../RulesActions";

describe("<RulesActions />", () => {
  const props = { createUrl: "/url/to/create/concept" };

  it("matches the latest snapshot", () => {
    const wrapper = shallow(<RulesActions {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
