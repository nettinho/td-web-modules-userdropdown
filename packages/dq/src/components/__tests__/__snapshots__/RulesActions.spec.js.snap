// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RulesActions /> matches the latest snapshot 1`] = `
<div
  style={
    Object {
      "float": "right",
    }
  }
>
  <Connect(Component) />
</div>
`;
